datastore/src/main/java/org/jboss/pnc/datastore/repositories/DeliverableAnalyzerLabelEntryRepositoryImpl.java

```diff
@@ -32,4 +33,12 @@ public class DeliverableAnalyzerLabelEntryRepositoryImpl
     public DeliverableAnalyzerLabelEntryRepositoryImpl() {
         super(DeliverableAnalyzerLabelEntry.class, Base32LongID.class);
     }
+
+    @Override
+    public DeliverableAnalyzerLabelEntry save(DeliverableAnalyzerLabelEntry entity) {
+        if (entity.getId() == null) {
+            entity.setId(new Base32LongID(Sequence.nextBase32Id()));
+        }
+        return super.save(entity);
```

Comment:
Another workaround I needed to use, otherwise would be used `AbstractRepository#save`, which would produce in null id generated exception.
Can you Honza try to explain to me why do we need this workaround exactly here? We use Base32LongId type for IDs also in BuildRecord, and there's no such problem. Why do we need this special handling here? Why doesn't handle it AbstractRepository alone?